src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt

```diff
@@ -2,15 +2,20 @@ package personal.opensrcerer.client.cache
 
 object SubsonicCache {
     private val cacheMap: MutableMap<String, SubsonicConfig>
+    private val url = System.getenv("sub_url") ?: "bonkersmusic.onthewifi.com"
```

Comment:
SupersonicConstants.getVariable(String)

Reply:
If I try to compile with this
`e: /buildapp/src/main/kotlin/personal/opensrcerer/client/cache/SubsonicCache.kt: (5, 23): Unresolved reference: SupersonicConstants`

Reply:
```    private val cacheMap: MutableMap<String, SubsonicConfig>
    private val url = SupersonicConstants.getVariable("SUB_URL") ?: "bonkersmusic.onthewifi.com"
    private val port = SupersonicConstants.getVariable("SUB_PORT").toIntOrNull() ?: 4040
    private val user = SupersonicConstants.getVariable("SUB_USER") ?: "Bonkers"
    private val pass = SupersonicConstants.getVariable("SUB_PASS") ?: "DNFWTF4201?!"
    private val version = SupersonicConstants.getVariable("SUB_VERSION") ?: "1.15.0"```

Reply:
You need to add `import personal.opensrcerer.launch.SupersonicConstants` in the file imports.